test/intergration/.env

```diff
@@ -1,4 +1,4 @@
 # for s3 provider
 AWS_ACCESS_KEY_ID=
 AWS_SECRET_ACCESS_KEY=
-AWS_REALM=eu-central-1
```

Comment:
@ky23 sorry accidentally deleted your question, I think AWS_REGION is unused and can be removed..not sure how it got in there...

Reply:
The tests are passed but there is a problem in the sonar configuration, 
I will fix it